libzkbob-rs-wasm/src/client/mod.rs

```diff
@@ -474,6 +496,41 @@ impl UserAccount {
         serde_wasm_bindgen::to_value(&data).unwrap()
     }
 
+    #[wasm_bindgen(js_name = "getTxInputs")]
+    /// Returns transaction inputs: account and notes
+    pub fn get_tx_inputs(&self, index: u64) -> Result<TxInput, JsValue> {
+        if index & constants::OUTPLUSONELOG as u64 != 0 {
```

Comment:
This check doesn't look correct. For example 8 & 7 == 0 too but it doesn't mean that 8 % 128 == 0. I guess we should use constants::OUT instead but, as for me, index % (constants::OUT + 1) == 0 looks much better even if it's a bit slower.

Reply:
It's definitely a mistake made by me. Of course the `constants::OUT` should be here instead `constants::OUTPLUSONELOG`. The speed doesn't make sense here because it isn't time-critical operation. But I believe the error message in the next line leaves no way to doubt about check semantic

Fixed in https://github.com/zkBob/libzkbob-rs/pull/46/commits/c21ea801b890216200e3ed539fa6ea13e707f70c